Resources/public/hbt/default.hbt

```diff
@@ -0,0 +1,29 @@
+<div class="content-view-line">
+    {{#each recommendations}}
+    <article class="class-article">
```

Comment:
if we except what we did in legacy, we don't really have any convention for that yet. And in any case, given the scope of this PR, `class-article` is fine for now.